src/app/banners/BannerWrapper.tsx

```diff
@@ -7,23 +7,38 @@ import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
 import FeaturesBanner from './FeaturesBanner';
 import { BannerManager } from './BannerManager';
 import { useEffect, useState } from 'react';
+import SubscriptionBanner from './SubscriptionBanner';
 
-const OFFER_END_DAY = new Date('2025-02-25');
+const OFFER_END_DAY = new Date('2025-03-17');
 
 const BannerWrapper = (): JSX.Element => {
   const [showBanner, setShowBanner] = useState<boolean>(false);
+  const [showSubscriptionBanner, setShowSubscriptionBanner] = useState<boolean>(false);
   const user = useSelector((state: RootState) => state.user.user) as UserSettings;
   const plan = useSelector<RootState, PlanState>((state) => state.plan);
 
   const bannerManager = new BannerManager(user, plan, OFFER_END_DAY);
```

Comment:
this will be reinstantiated in every render, remember to use useMemo to avoid this undesired behaviour